docs/book/user-guide/starter-guide/track-ml-models.md

```diff
@@ -0,0 +1,355 @@
+* `archived`: This is archived and no longer relevant. This stage occurs when a model moves out of any other stage.
+
+{% tabs %}
+{% tab title="Python SDK" %}
+```python
+from zenml.model import ModelVersion
+
+# Get latest model version
+model_version = ModelVersion(
+    name="iris_classifier",
+    version="latest"
+)
+
+# Get a model from a version
+model_version = ModelVersion(
+    name="iris_classifier",
+    version="my_version",
+)
+
+# Pass the stage into the version field
+# to get the model by stage
+model_version = ModelVersion(
+    name="iris_classifier",
+    version="staging",
+)
+
+# This will set this version to production
+model_version.set_stage(stage="production", force=True)
+```
+{% endtab %}
+
+{% tab title="CLI" %}
+```shell
+# List staging models
+zenml model version list <MODEL_NAME> --stage staging 
+
+# Update to production
+zenml model version update <MODEL_NAME> <MODEL_VERSIONNAME> -s production 
+```
+{% endtab %}
+{% tab title="Cloud (Dashboard)" %}
+The [ZenML Cloud](https://zenml.io/cloud) dashboard has additional capabilities, that include easily changing the stage:
+
+![ZenML Cloud Transition Model Stages](../../.gitbook/assets/dcp_transition_stage.gif)
+
+{% endtab %}
+{% endtabs %}
+
+## Facilitating Artifacts Exchange Between Pipelines Using MCP
+
+A ZenML Model spans multiple pipelines, and is a key concept that brings disparate pipelines together. A simple example is illustrated below:
+
+<figure><img src="../../.gitbook/assets/mcp_pipeline_overview.png" alt=""><figcaption><p>A simple example of two pipelines interacting between each other.</p></figcaption></figure>
+
+Each time the `train_and_promote` pipeline runs, it creates a new `iris_classifier`. However, it only promotes the created model to `production` if a certain accuracy threshold is met. The `do_predictions` pipeline simply picks up the latest promoted model and runs batch inference on it. That way these two pipelines can independently be run, but can rely on each other's output.
+
+One way of achieving this is to fetch the model directly in your step:
+
+```python
+from zenml import step, get_step_context
+
+# IMPORTANT: Cache needs to be disabled to avoid unexpected behavior
+@step(enable_cache=False)
+def predict(
+    data: pd.DataFrame,
+) -> Annotated[pd.Series, "predictions"]:
+    # model_name and model_version derived from pipeline context
+    model_version = get_step_context().model_version
+
+    # Fetch the model directly from the model control plane
+    model = model_version.get_model_artifact("trained_model")
+
+    # Make predictions
+    predictions = pd.Series(model.predict(data))
+    return predictions
+```
+
+However, this approach has the downside that if the step is cached, then it could lead to unexpected results. You could simply disable the cache in the above step or corresponding pipeline. However, one other way of achieving this would be to resolve the artifact at the pipeline level:
```

Comment:
```suggestion
However, this approach has the downside that if the step is cached, then it could lead to unexpected results. You could simply disable the cache in the above step or the corresponding pipeline. However, one other way of achieving this would be to resolve the artifact at the pipeline level:
```